index.js

```diff
@@ -17,22 +17,22 @@ const cards = [
     text: 'Booklist of books that I have read and enjoyed. I have also read a lot of books that I have not enjoyed. Here I can manage my books and read them later.',
     img: 'img1',
     src: 'assets/awesome_books.png',
-    tech: ['html', 'bootstrap', 'Ruby'],
+    tech: ['html', 'bootstrap', 'Js'],
     links: {
       live: "'https://awesomebooks-1.netlify.app'",
       source: "'https://github.com/mrEckendonk/Awesome-books'",
     },
   },
   {
-    title: 'Data Dashboard Healthcare',
-    intro: 'World event about Renewable Energy, Solar Energy, Clean Energy, with fantastic speakers and workshops',
-    text: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    title: 'Todo List Application',
+    intro: 'EMPTY',
+    text: 'A nice and easy to use todo list application. You can add, delete, edit and mark tasks as done. Never forget to do your tasks!',
     img: 'img2',
-    src: 'assets/image_backgrnd2.png',
-    tech: ['html', 'bootstrap', 'Ruby'],
```

Comment:
Nice job with the contrast to increase accessibility!

Reply:
Thanks!